docs/modules/ROOT/pages/how-tos/opsgenie.adoc

```diff
@@ -0,0 +1,263 @@
+= Integrating Alertmanager with OpsGenie
+
+This how-to assumes that you're already using OpsGenie.
+It additionally assumes that you're familiar with managing schedules and escalations.
+
+== Prerequisites
+
+In order to setup Alertmanager integration with OpsGenie, you'll need sufficient permissions in OpsGenie to setup integrations and heartbeats.
+If you're using teams in OpsGenie, request role "admin" in your team to be able to follow along.
+
+== Configuring the integration in OpsGenie
+
+In OpsGenie, navigate to "Teams > Your Team > Integrations" and enable the Prometheus and REST API integrations.
+You can use the same integrations to receive alerts from multiple Alertmanagers.
+Save the API keys which are generated when enabling the integrations in Vault, we'll reference them from the Commodore hierarchy.
```

Comment:
Please provide the path where to put those whalues. It is mentioned down below but having to jump back and forth is anoying.